src/fsrs/algorithm.ts

```diff
@@ -159,16 +158,25 @@ export class FSRSAlgorithm {
     return this.apply_fuzz(newInterval, elapsed_days)
   }
 
+  /**
+   * @see https://github.com/open-spaced-repetition/fsrs4anki/issues/697
+   */
+  linear_damping(delta_d: number, old_d: number): number {
+    return +((delta_d * (10 - old_d)) / 9).toFixed(8)
```

Comment:
Remove unnecessary parentheses.

```suggestion
    return +(delta_d * (10 - old_d) / 9).toFixed(8)
```